server/routes/screenshot/html.py

```diff
@@ -27,6 +31,48 @@
 bp = flask.Blueprint("screenshot", __name__, url_prefix='/screenshot')
 
 
+@bp.route('/')
```

Comment:
Can you add a comment description here & a description of the url param options?

Reply:
Done